Meadowlark-js/backends/meadowlark-elasticsearch-backend/docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.8"
+
+volumes:
+  esdata01:
+    driver: local
+
+networks:
+  default:
+    name: elastic
+    external: false
+
+services:
+  elasticsearch-node1:
+    image: docker.elastic.co/elasticsearch/elasticsearch:latest
```

Comment:
We should be using a specific version instead of the latest to have control of the changes, version added here should also be updated in https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/pull/257/files#diff-164eb150e4c9ced233eb67c4faedaf510d3bf88c88af71e5a9b7f20a189daa51R13

Reply:
Version latest is not available here:
https://www.docker.elastic.co/r/elasticsearch

Which leads to this error message:
![image](https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/assets/5459528/f1cb368d-8f03-429d-b141-b359a30e85b6)


Also, this `docker.elastic.co/elasticsearch/elasticsearch` refers to the image in the elastic search docker registry, but we could use the one for docker hub `elasticsearch` https://hub.docker.com/_/elasticsearch

Reply:
Either location is fine with me, so long as we pin to a specific version using the SHA256 hash.

Tip: https://stackoverflow.com/questions/32046334/where-can-i-find-the-sha256-code-of-a-docker-image